packages/libs/snap/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@kadena/snap",
+  "version": "1.0.0",
+  "description": "MetaMask snap to manage your Kadena accounts and KDA.",
+  "license": "BSD-3-Clause",
+  "main": "./dist/bundle.js",
+  "files": [
+    "dist/",
+    "images/",
+    "snap.manifest.json"
+  ],
+  "scripts": {
+    "build": "mm-snap build",
+    "build:clean": "pnpm clean && pnpm build",
+    "clean": "rimraf dist",
+    "lint": "pnpm lint:eslint && pnpm lint:misc --check",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:fix": "pnpm lint:eslint --fix && pnpm lint:misc --write",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path .gitignore",
+    "serve": "mm-snap serve",
+    "start": "mm-snap watch",
+    "test": "jest --detectOpenHandles"
+  },
+  "dependencies": {
+    "@kadena/cryptography-utils": "workspace:*",
+    "@metamask/snaps-sdk": "^3.1.1",
+    "@metamask/snaps-ui": "^2.0.0",
+    "bip39": "^3.1.0",
```

Comment:
We're not using `@kadena/hd-wallet`. I think that's because we're relying on the metamask logic to generate keypairs? Do we use the same derivation path here as we do in `@kadena/hd-wallet`?